File: Inlezen.py
def inlezen():   
    data=open("RNA-Seq-counts.txt", 'r')
    annotatie=open("WCFS1_anno.txt", 'r')
    
    data_list=[]
    anno_list=[]
    
    for line in data:
        data_list.append(line)
    for line in annotatie:
        anno_list.append(line)
    

    print (data_list)

def main():
    inlezen()

main()

File: project11.venv/inlezen.R
#Inlezen van het bestand
data = read.delim("../Blok11/data/RNA-Seq-counts.txt", header = TRUE, sep = "\t", comment.char = "#")

gene_ID = data[,1]
selected = gene_ID[100:120]
print(selected)


write.csv(selected, file="ID.csv", row.names = F)


